charts/internal/gvisor/templates/runtimeclass-gvisor.yaml

```diff
@@ -1,4 +1,8 @@
+{{- if semverCompare ">= 1.20-0" .Capabilities.KubeVersion.GitVersion }}
```

Comment:
Few lines below `.Values.config.kubernetesVersion` is used to compare the versions.

Can we make the comparisons consistent - maybe use `Values.config.kubernetesVersion` as we anyways pass it already?

Reply:
> Few lines below .Values.config.kubernetesVersion is used to compare the versions.

Isn't it better to use the helm "built-in" function?

Reply:
Ok, let's keep it as it is. Let's leave the clean up for https://github.com/gardener/gardener-extension-provider-aws/issues/595 as maybe there are also other things that have to be removed.